lib/ex_cell/adapters/cell_js.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ExCell.Adapters.CellJS do
+  @moduledoc false
+
+  @behaviour ExCell.Adapter
+
+  alias Phoenix.HTML.Tag
+
+  def data_attribute(name, data \\ [], params \\ %{})
+  def data_attribute(name, nil, params), do: data_attribute(name, [], params)
+  def data_attribute(name, data, params) when is_list(data), do:
+    Keyword.merge(data, cell: name, cell_params: Poison.encode!(params))
+
+  def attributes(name, attributes \\ [], params \\ %{}) do
+    Keyword.put(
+      attributes,
+      :data,
+      data_attribute(name, Keyword.get(attributes, :data), params)
+    )
+  end
+
+  def container(%{
+    name: name,
+    attributes: attributes,
+    params: params,
+    content: content
+  }) do
+    {tag, attributes} = Keyword.pop(attributes, :tag, :div)
+    {closing_tag, attributes} = Keyword.pop(attributes, :closing_tag, true)
+
+    attributes = attributes(name, attributes, params)
+
+    case closing_tag do
```

Comment:
Just curious, why not check for `content` here?

```ex
case content do
  nil -> Tag.tag(tag, attributes)
  _ -> Tag.content_tag(tag, content, attributes)
end
```

In other words, do we need `closing_tag`?

Reply:
@sn3p we do, because there are only a small number of tags that don't need a closing tag. Phoenix doesn't automatically detect these.